_posts/0000-01-02-RenataUhlmann.md

```diff
@@ -0,0 +1,11 @@
+___
+
+layout: slide
+
+title: "Welcome to our second slide!"
+
+___
+
+Way to go!
```

Comment:
I don't think this is a good title. plus the line 9 should have a special approach